jupytext/__init__.py

```diff
@@ -16,10 +16,6 @@
     TextFileContentsManager = reraise(err)
 
 
-def _jupyter_labextension_paths():  # pragma: no cover
-    return [{"src": "labextension", "dest": "jupyterlab-jupytext"}]
-
-
 def load_jupyter_server_extension(app):  # pragma: no cover
```

Comment:
This is a server extension. Should it remain in `jupytext`, or go to the `jupyterlab-jupytext` package?

Reply:
So it should be moved to `jupyterlab-jupytext`, unless I am misunderstanding something here.

Reply:
Actually in my comment I was referring to `load_jupyter_server_extension`. I am wondering if we could/should move it to the `jupyterlab-jupytext` extension, along with the one above which you have already moved.